apps/studio/src/shared/lib/dialects/clickhouse.ts

```diff
@@ -74,6 +74,14 @@ export const ClickHouseData: DialectData = {
     const matched = value.match(UNWRAPPER);
     return matched ? matched[1] : value;
   },
+  importDataType: {
+    stringType: 'String',
```

Comment:
You'll probably need to check the length of the strings, if > 255 I'd make this a TEXT rather than a varchar. Also what varchar length do you use by default? That should probably be explicit

Reply:
I can add a length check to determine which is which. I'll add another "string" type so I can explicitly say which one to use based on connection type. I use the varchar default per connection, but I can go back and get more specific. It's different per connection